tests/e2e/conftest.py

```diff
@@ -141,17 +144,28 @@ def factory(
             kind="secret", namespace=config.ENV_DATA["cluster_namespace"]
         )
         secrets = [
-            "noobaa-root-master-key",
             "noobaa-admin",
             "noobaa-operator",
             "noobaa-db",
             "noobaa-server",
             "noobaa-endpoints",
         ]
+
         if is_kms_enabled():
             secrets = [
                 secret for secret in secrets if secret != "noobaa-root-master-key"
             ]
+
+        if (
+            version.get_semantic_ocs_version_from_config() >= version.VERSION_4_14
+            and not config.DEPLOYMENT.get("kms_deployment")
+        ):
+            secrets.extend(
+                ["noobaa-root-master-key-backend", "noobaa-root-master-key-volume"]
+            )
+        elif not config.DEPLOYMENT.get("kms_deployment"):
+            secrets.append("noobaa-root-master-key")
+
```

Comment:
This code can removed. The kms issue is already fixed with https://github.com/red-hat-storage/ocs-ci/pull/8591 and https://github.com/red-hat-storage/ocs-ci/pull/7523

Reply:
@PrasadDesala #8591 is bit confusing.. I understand you're taking out `noobaa-root-master-key` for non kms cluster. but for 4.14 onwards there are two different secrets for kms, key-volume and key-backend

Reply:
#7523 is only specific to test_noobaa_rebuild. its not relevant to this test right